### DISCUSSION TITLE
Integrate weblate into git-flow branching model

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#8531] [Q&A]
How do I integrate the actions of weblate into a git-flow branching model?
I can't allow weblate to create PR's against main or develop.

My feeling is to disallow weblate to generate automatic PRs and just trigger PRs manually when I find it the right time.

How do you (other users) integrate it in your branching model?

**Comment by nijel** [comment]
You can configure push branch in the component configuration, if that's what you're looking for (sorry, I have no clue how git-flow works).
